test/index.js

```diff
@@ -101,24 +116,29 @@ describe('Schwifty', () => {
             ]
         });
 
-        getServer(config, (err, server) => {
+        getServer(config).then(function (server) {
 
-            expect(err).to.not.exist();
             expect(server.models().Dog.$$knex).to.not.exist();
             expect(server.models().Person.$$knex).to.not.exist();
 
-            server.initialize((err) => {
+            return server;
 
-                expect(err).to.not.exist();
-                expect(server.models().Dog.$$knex).to.exist();
-                expect(server.models().Person.$$knex).to.exist();
+        }).then(initializeServer).then(function(server) {
+
+              expect(server.models().Dog.$$knex).to.exist();
+              expect(server.models().Person.$$knex).to.exist();
+
+              done();
+
+        }).catch((error) => {
+
+            console.log("First test error", error);
+            done(error);
 
-                done();
-            });
         });
     });
 
-    it('tears-down connections onPostStop.', (done) => {
+    /*it('tears-down connections onPostStop.', (done) => {
```

Comment:
Rather than commenting all other tests, you can also turn `it(/* ... */)` to `it.only(/* ... */)` on the one test you'd like to run.

Reply:
👍 🙏 💋